### ISSUE TITLE
Restore correct classes to lists and tuples

### ISSUE THREAD

**Issue by Rizhiy** [makinacorpus/easydict#9]
Currently, all tuples, lists and their subclasses are converted to lists, which is not ideal.
They should be converted back to their class/subclass.

**Comment by loveorchids** [comment]
Agreed, in some cases/libraries, input parameters type is fixed, which requires people to convert the easydict item back to whatever the function needs, seems to be a little bit awkward.

**Comment by leplatrem** [comment]
Don't hesitate to open a PR and fix it:

https://github.com/makinacorpus/easydict/blob/52eb7b31357a815f9b96a46a176e992aa1c5dfed/easydict/__init__.py#L125-L127
